configure.ac

```diff
@@ -2,7 +2,7 @@
 # Process this file with autoconf to produce a configure script.
 
 AC_PREREQ([2.65])
-AC_INIT([tox], [0.0.5])
+AC_INIT([tox], [0.1.0])
 AC_CONFIG_AUX_DIR(configure_aux)
```

Comment:
Just to keep in mind.. if it's the "first" release you can probably leave it at 0:0:0, but make sure not to forget the LIBTOXCORE_LT_VERSION setting in configure.ac in future releases, it defines the library compatibility (this is passed to libtool as parameter) and has influence on the name of the library, i.e. .so.x.y.z,  I think cmake also has some setting for this.